### PR TITLE
Prioritize exposes tab order? #210

### PR DESCRIPTION
**Pull request by timoline**

### CODE REVIEW

**Comment by timoline** [comment]
o damn , one space....think its easier to fix for you then for me

File: src/components/device-page/index.tsx [hunk 1]
```diff
@@ -57,7 +57,7 @@ export class DevicePage extends Component<DevicePageProps & GlobalState, {}> {
             <h3>{device.friendly_name}</h3>
             <ul className="nav nav-tabs">
```
Comment (nurikk) [inline src/components/device-page/index.tsx:58]: you only need change here, don't touch any other places.
Kindly follow this order:

About, Exposes, Bind, Reporting, State, Clusters

File: src/components/device-page/index.tsx [hunk 2]
```diff
@@ -18,7 +18,7 @@ type UrlParams = {
 type DevicePageProps = RouteComponentProps<UrlParams>;
 
 
-type TabName = "info" | "bind" | "state" | "exposes" | "clusters" | "reporting";
+type TabName = "exposes" | "reporting" | "bind" | "info" | "state" | "clusters" ;
```
Comment (nurikk) [inline src/components/device-page/index.tsx:21]: No need change here

File: src/components/device-page/index.tsx [hunk 3]
```diff
@@ -27,18 +27,18 @@ export class DevicePage extends Component<DevicePageProps & GlobalState, {}> {
         const device = devices.get(dev);
 
         switch (tab) {
-            case "info":
-                return <DeviceInfo device={device} />;
+            case "exposes":
```
Comment (nurikk) [inline src/components/device-page/index.tsx:30]: No need change here

**Review by nurikk — CHANGES_REQUESTED** [review]

**Comment by timoline** [comment]
Ok

**Pull request closed without merging**